src/xpath.jison

```diff
@@ -104,6 +105,16 @@ filter_expr:  base_expr predicate     { $$ = new xpathmodels.XPathFilterExpr({ex
                                         $$ = filterExpr; }
         ;
 
+hashtag_expr:  HASH QNAME SLASH hashtag_path              { $$ = new xpathmodels.HashtagExpr({initial_context: xpathmodels.XPathInitialContextEnum.HASHTAG,
```

Comment:
Using QNAME instead of step here as I don't think that we have filters in hashtag expressions at the moment.